pkg/scanners/trivy/trivy.go

```diff
@@ -89,11 +90,17 @@ func main() {
 	flag.Parse()
 	ctx := context.Background()
 
+	var err error
+
 	if err := logger.Configure(); err != nil {
 		fmt.Fprintln(os.Stderr, "error setting up logger:", err)
 		os.Exit(generalErr)
 	}
 
+	if trivylogger.Logger, err = trivylogger.NewLogger(false, true); err != nil {
```

Comment:
looks like this disables trivy's logger, so we won't get those logs. can we replace `trivylogger.Log` with our own logger or create a new zap logger?

something like
```
	logger, _ := zap.NewProduction()
	sugar := logger.Sugar()
	trivylogger.Logger = sugar
```

Reply:
changed! I don't think there are any other logs from trivy right now besides the the scan output though. Is there a logs output we are looking for?

Reply:
yes, just the trivy logs and the scanner logs